packages/eas-json/src/errors.ts

```diff
@@ -5,3 +5,5 @@ export class MissingEasJsonError extends Error {}
 export class MissingProfileError extends Error {}
 
 export class MissingParentProfileError extends Error {}
+
+export class InvalidResourceClassError extends Error {}
```

Comment:
Why is this error class in `eas-json` package? If we decided to go for the CLI param, then it has nothing to do with eas.json.

Reply:
I have no good answer for this 😅 I just grep'd for *Error and figured this is where error definitions go...

I'll remove this since we're not using it anymore.